.github/workflows/publish.yml

```
@@ -0,0 +1,53 @@
+name: Release Pipeline (develop/2.0)
```

maybe mention it's npm not the extension itself
```suggestion
name: NPM Release Pipeline (develop/2.0)
```